src/modules/form/components/viewable/DynamicViewField.tsx

```diff
@@ -115,8 +115,12 @@ const DynamicViewField: React.FC<DynamicViewFieldProps> = ({
         <TextContent
           {...commonProps}
           value={
-            [TRUE_OPT, FALSE_OPT].find((o) => o.code === String(value))
-              ?.label || <NotCollectedText variant='body2' />
+            [TRUE_OPT, FALSE_OPT].find((o) => {
+              return (
+                o.code === String(value) ||
+                String(o.numericCode) === String(value)
```

Comment:
I'm not sure about this. "numeric code" was introduced as a way to attach a different "score" to each pick list option. These are simple yes/no's, they don't need a score associated with it.

This is a downstream effect of the fact that External Forms collect boolean values as `1/0`, while HMIS forms collect them as `true/false`. I think it would be better to normalize that behavior instead. (Either adjusting External Forms to collect as booleans, or normalize values before storing them in form processor?).